CITATION.cff

```diff
@@ -648,6 +648,10 @@ authors:
   family-names: Gordon-McKeon
   alias: shaunagm
 
+- given-names: 
```

Comment:
```suggestion
- given-names: Mel
```
Adding this back since I accidentally deleted it when resolving a merge conflict. 😅